README.md

```diff
@@ -37,3 +37,18 @@ When you run `cargo-temp` for the first time it will be created automatically
 * `temporary_project_dir`: path where the temporary projects are created (cache directory by default).
 * `cargo_target_dir`: cargo's target directory override (unset by default).
    This setting is ignored if `CARGO_TARGET_DIR` is already set.
+* `editor` and `editor_args`: path to an IDE to start instead of a shell and arguments for it (unset by default).
+
+  Example config to run VS Code on Unix:
```

Comment:
```suggestion
### Example config to run VS Code on Unix:
```

Reply:
Then the example should include `temporary_project_dir` while at the moment it explains only how to configure `editor` and `editor_args` to run VS Code. By the way, should `temporary_project_dir` be explicitly stored in the config? I don't think so since it points to a well known place which can be detected on the fly if not specified explicitly.

Reply:
> Then the example should include `temporary_project_dir` while at the moment it explains only how to configure `editor` and `editor_args` to run VS Code.

Nah that's fine. I expect the user will understand, it's kinda obvious.


> By the way, should `temporary_project_dir` be explicitly stored in the config? I don't think so since it points to a well known place which can be detected on the fly if not specified explicitly.

Yes... it was just a way to create the config file with the most wanted parameters set already. I think it's more user-friendly.

:thinking: if that detection ever changed it would be annoying that the path change. :thinking: actually it's better how it's done at the moment.

@yozhgoor do you have an opinion on this? What makes more sense to you?

Reply:
I'm not really sure to understand.
I don't think `temporary_project_dir` need to be in the example about the editor config. 
But in the config it's needed for the users who want to change where the temporary directory is created.

Reply:
It's not "needed" as you can detect it (which is what you do for the default value). So you could detect it always if not specified in the config.

Reply:
The config can contain commented options, so it won't be fully empty on first start and no need to go to readme in the repo.  For example:

```toml
# Path where the temporary projects are created (cache directory by default).
# temporary_project_dir = "~"

# Cargo's target directory override (unset by default). This setting is ignored if CARGO_TARGET_DIR is already set.
# cargo_target_dir = "./target"
```

Reply:
I don't think this will change a lot the behavior of cargo-temp. But like Cecile said, the actual explicit way is more user friendly especially for users who don't really know about directory structure's standards.

Reply:
Thanks for the example, things are more clearer

Reply:
Okey, sorry for my misunderstand.
I mean i don't think we need to have a long config file with explanations because we don't have a lot of settings in it.